controllers/articles.js

```diff
@@ -0,0 +1,17 @@
+const {findArticle} = require('../models/articles')
+
+function getArticle(req,res,next) {
+  const {article_id} = req.params
+  return findArticle(article_id).then(({rows})=>{
+    if(!rows[0]){
```

Comment:
Error handling should be dealt with by the model and handled with a rejection that we pick up in the catch block below and pass to the next error handlers.